camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -245,23 +246,16 @@ def _get_corners(img, board, refine = True, checkerboard_flags=0):
 
     return (ok, corners)
 
-def _get_charuco_corners(img, board, refine):
+def _get_charuco_corners(img, board):
     """
     Get chessboard corners from image of ChArUco board
     """
-    h = img.shape[0]
-    w = img.shape[1]
-
-    if len(img.shape) == 3 and img.shape[2] == 3:
-        mono = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
-    else:
-        mono = img
 
```

Comment:
you can remove then also the additional empty line here
I guess then cv2.aruco.detectMarkers supports rgb images as well and not just mono as before? Did you check this?

Reply:
Documentation doesn't mention it https://docs.opencv.org/3.4/d9/d6a/group__aruco.html#ga061ee5b694d30fa2258dd4f13dc98129:~:text=%E2%97%86-,detectMarkers(),-void%20cv%3A%3Aaruco
I've removed it becaused unused